### PR TITLE
Use matrix of constraints

### PR DESCRIPTION
**Pull request by blegat**
Requires https://github.com/jump-dev/MathOptInterface.jl/pull/1287

### CODE REVIEW

**Comment by joaquimg** [comment]
Looks like a great code improvement!
Have you done MOIPaper like benchmarks?

**Comment by blegat** [comment]
Not yet but we can expect some improvements with the latest changes on MOI

File: src/MOI_wrapper/MOI_wrapper.jl [hunk 1]
```diff
@@ -380,7 +291,33 @@ function MOI.copy_to(
...
+)
+    _copy_to(dest, src)
+    return MOI.Utilities.identity_index_map(src)
+end
+function MOI.copy_to(
+    dest::Optimizer,
+    src::MOI.Utilities.UniversalFallback{Model};
+    copy_names::Bool = false
+)
+    return MOI.copy_to(dest, src.model)
```
Comment (odow) [inline src/MOI_wrapper/MOI_wrapper.jl:309]: Doesn't this just ignore any extra attributes in `src`?

**Comment by odow** [comment]
Close in favor of #114

**Pull request closed without merging**